### DISCUSSION TITLE
Check out `Terminal-Icons`

### DISCUSSION THREAD

**Discussion by tig** [tui-cs/Terminal.Gui#2611] [Ideas]
https://github.com/devblackops/Terminal-Icons

@tznind can we make the file services use the same icons?

**Comment by tznind** [comment]
Definetly, this is exactly what `FileDialogStyle.IconGetter` is intended for.

All we would need is the extension/name->unicode number map and for user to have the fonts installed.

Could make it a ConfigurationManger setting quite easily too.

**Comment by tznind** [comment]
Here is a quick stab https://github.com/gui-cs/Terminal.Gui/pull/2613

If you can convert this to an issue I can link it to the above PR.
